java.lang.ClassCastException: org.codehaus.groovy.runtime.GStringImpl cannot be cast to java.lang.String

I've just noticed that version `0.8.0` fails my project's build:
```
Caused by: java.lang.ClassCastException: org.codehaus.groovy.runtime.GStringImpl cannot be cast to java.lang.String
	at com.vanniktech.maven.publish.MavenPublishPom$Companion.fromProject(MavenPublishPom.kt:32)
	at com.vanniktech.maven.publish.BaseMavenPublishPlugin.apply(BaseMavenPublishPlugin.kt:12)
	at com.vanniktech.maven.publish.BaseMavenPublishPlugin.apply(BaseMavenPublishPlugin.kt:7)
	at org.gradle.api.internal.plugins.ImperativeOnlyPluginTarget.applyImperative(ImperativeOnlyPluginTarget.java:42)
	at org.gradle.api.internal.plugins.RuleBasedPluginTarget.applyImperative(RuleBasedPluginTarget.java:50)
	at org.gradle.api.internal.plugins.DefaultPluginManager.addPlugin(DefaultPluginManager.java:177)
	at org.gradle.api.internal.plugins.DefaultPluginManager.access$300(DefaultPluginManager.java:51)
	at org.gradle.api.internal.plugins.DefaultPluginManager$AddPluginBuildOperation.run(DefaultPluginManager.java:267)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:402)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:394)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$1.execute(DefaultBuildOperationExecutor.java:165)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:250)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:158)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:92)
	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
	at org.gradle.api.internal.plugins.DefaultPluginManager$2.execute(DefaultPluginManager.java:155)
	at org.gradle.api.internal.plugins.DefaultPluginManager$2.execute(DefaultPluginManager.java:152)
	at org.gradle.configuration.internal.DefaultUserCodeApplicationContext.apply(DefaultUserCodeApplicationContext.java:48)
	at org.gradle.api.internal.plugins.DefaultPluginManager.doApply(DefaultPluginManager.java:152)
	... 138 more
```

BTW, I am using Android Studio 3.5.3 with Gradle 5.4.1. Do you have the same error?

--- a/src/main/kotlin/com/vanniktech/maven/publish/MavenPublishPom.kt
+++ b/src/main/kotlin/com/vanniktech/maven/publish/MavenPublishPom.kt
@@ -47,7 +47,7 @@ internal data class MavenPublishPom(
     )
 
     private fun findMandatoryProperty(project: Project, propertyName: String): String {
-      val value = project.findProperty(propertyName) as String?
+      val value = project.findProperty(propertyName)?.toString()
       if (value == null) {
         throw IllegalArgumentException("Please define \"$propertyName\" in your gradle.properties file")
       } else {

